doc/glossary.rst

```diff
@@ -344,11 +344,11 @@ General Concepts
         likelihoods.
 
     estimator tags
-        A proposed feature (e.g. :issue:`8022`) by which the capabilities of an
-        estimator are described through a set of semantic tags.  This would
-        enable some runtime behaviors based on estimator inspection, but it
-        also allows each estimator to be tested for appropriate invariances
-        while being excepted from other :term:`common tests`.
+    	Estimator tags are tags that show the capabilities of an estimator.
+    	They were proposed in :issue:`8022`. From the tags, runtime behaviors can
+    	be enabled based on estimator inspection. Each estimator can also be 
+    	tested for appropriate invariances while being excepted from other 
+    	:term:`common tests`.
 
         Some aspects of estimator tags are currently determined through
```

Comment:
I think this is no longer appropriate, although `_estimator_type` and `_pairwise` have not yet been ported to the new tags framework.

Reply:
Thanks. I’ll cancel this pull request.  Basically, what’s needed in the glossary page is a brief explanation of the estimator tag, and then provide a link to the developer’s page?